devicetypes/qubino/qubino-dimmer.src/qubino-dimmer.groovy

```diff
@@ -114,7 +114,7 @@ private getINPUT_TYPE_TEMPERATURE_SENSOR() {3}
 def installed() {
 	// Device-Watch simply pings if no device events received for 32min(checkInterval)
 	sendEvent(name: "checkInterval", value: 2 * 15 * 60 + 2 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
-
+	setInitialConfiguration()
```

Comment:
Why not just have `setInitialConfiguration()` and `readConfigurationFromTheDevice()` return the commands to send (instead of calling `sendHubCommand`) and return those commands from `installed`?

Reply:
Actually, these seem like things that should be part of `configure()`.

Reply:
Your are totally right. I moved that part to `configure()`

Reply:
I'd prefer to leave readConfigurationFromTheDevice() as a separate method, because it's a part of our template code for handling device preferences.

Reply:
@KKlimczukS I understand that. You can leave it as a separate method but change the way that it works. This will keep all of the commands to the device together and they will get sent together. Then you'd have:
```
def configure() {
...
    cmds += readConfigurationFromTheDevice()
...
    return cmds
}
```

And any time outside of a capability command function you would just do:
```
sendHubCommand(readConfigurationFromTheDevice())
```

Reply:
In the past we've had issues with Z-Wave devices where if we started sending them data from the DTH too quickly (usually from within `installed()`) they wouldn't be in a state where they would act on it properly. Having those commands in `configure()` or in `zwaveEvent(WakeUpNotification)` helped. (This is more of a heads up since you use the template elsewhere.)

Reply:
I would also do similar with `syncConfiguration()` and how it is used in `updated()`. I see that is the only thing in `updated()` here, though that isn't the case here: https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/37229/files#diff-c4d3d4cb621c024a3943efd31d82d407R176

Reply:
@dkirker I see Your point. 
readConfigurationFromTheDevice() was moved (and renamed) to configure().